github/orgs_projects.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ListProjectsOptions specifies the optional parameters to the
+// OrganizationsService.ListProjects method.
+type ListProjectsOptions struct {
```

Comment:
Yeah, no problem. Happy to change it.

I wasn't sure which name to use... it seems like there are some that use both naming conventions...

Some examples:
repos_commits.go:90:type CommitsListOptions struct {
orgs_members.go:49:type ListMembersOptions struct {

I'll change the name to `ProjectsListOptions` to match the convention used by `CommitsListOptions`

Do you have a preference for where it should be defined? Since both `orgs_projects.go` and `repos_projects.go` will be using it?